# HG changeset patch
# User Rik <rik@octave.org>
# Date 1463377012 25200
#      Sun May 15 22:36:52 2016 -0700
# Node ID e316b1548d2db20fb1596f6802a425a33848193f
# Parent  4c080cbc4ef946449deec8872e2a39bc6ea9633a
isdir.m: Don't call print_usage() with an argument.

* isdir.m: Don't call print_usage with the argument of "isdir".

diff --git a/scripts/general/isdir.m b/scripts/general/isdir.m
--- a/scripts/general/isdir.m
+++ b/scripts/general/isdir.m
@@ -18,18 +18,19 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {} {} isdir (@var{f})
 ## Return true if @var{f} is a directory.
 ## @seealso{exist, stat, is_absolute_filename, is_rooted_relative_filename}
 ## @end deftypefn
 
 function retval = isdir (f)
+
   if (nargin != 1)
-    print_usage ("isdir");
+    print_usage ();
   endif
 
   ## Exist returns an integer but isdir should return a logical.
   retval = (exist (f, "dir") == 7);
 
 endfunction
 
 
